conf/nginx/via/direct_proxy.conf

```diff
@@ -0,0 +1,21 @@
+set $upstream $proxied_uri$is_args$args;
+proxy_ssl_server_name on;
+proxy_pass $upstream;
+proxy_redirect ~^(.*)$ $original_scheme://$http_host/proxy/static/$1;
```

Comment:
This line may not be needed in the `location ~ /proxy/static/(?<proxied_uri>.*)` now (only in the `location @handle_redirect`) since, if it is a redirect, it's gonna hit the `error_page 301 302 307 = @handle_redirect` line?

Reply:
Maybe? I'd need to test it, but seems likely